ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -114,7 +114,7 @@ const HeaderInfo = () => {
           alignSelf: 'flex-end',
         }}
       >
-        <Tooltip position="bottom" title={t('accountDetails')} interactive>
```

Comment:
This was also breaking an e2e test. I'm wondering if someone on @MetaMask/confirmations can help me understand this

Reply:
Which test and how was it breaking?

Reply:
test suite is: `test/e2e/tests/confirmations/signatures/permit.spec.ts`
failing test is on line 33:
```
describe('Confirmation Signature - Permit @no-mmi', function (this: Suite) {
  it.only('initiates and confirms and emits the correct events', async function () {
```
Change is to remove interactive prop from Tooltip component in `ui/pages/confirmations/components/confirm/header/header-info.tsx`

Recording:

https://github.com/user-attachments/assets/74110e64-8aed-4a0a-b891-649c1f5b0225

Reply:
Incorporated the fix from this PR: https://github.com/MetaMask/metamask-extension/pull/27450